examples/readme.md

```diff
@@ -11,15 +11,15 @@ git clone https://github.com/pagantis/orders-api-client.git .
 composer install
 ```
 
-Then you can run the web application using PHP's built-in server:
+Then you can run the web application using PHP's built-in server via Composer Scripts:
 
 ```
-php -S 0.0.0.0:8000 -t examples/
+composer run-script examples-server
 ```
 
-The web application is running at [http://localhost:8000/simpleTest.php](http://localhost:8000/simpleTest.php).
+The web application is running at [http://localhost:8000/index.php](http://localhost:8000/index.php).
```

Comment:
Entering in this site I receive this message on screen:
Warning: require_once(../examples/utils/Helpers.php): failed to open stream: No such file or directory in /Users/pgarces/Project/libraries/ordersApiClient/examples/index.php on line 7

Fatal error: require_once(): Failed opening required '../examples/utils/Helpers.php' (include_path='.:/usr/local/Cellar/php@7.2/7.2.23/share/php@7.2/pear') in /Users/pgarces/Project/libraries/ordersApiClient/examples/index.php on line 7

Reply:
fixed path